packages/demo/components/pages/apis/transaction/builder/readOnlyTxInReference.tsx

```diff
@@ -5,7 +5,7 @@ export default function ReadOnlyTxInReference() {
   return (
     <Section
       sidebarTo="readOnlyTxInReference"
-      header="Set read only reference input"
+      header="Set a read-only reference input"
```

Comment:
For this, same reason above. If want to make sure its grammatically correct i would prefer makes all flags as plural, and not favouring a single change.

Reply:
as you prefer (but the hyphen "-" is important grammatically and for clarity)